waltid-services/waltid-service-commons/src/main/kotlin/id/walt/commons/config/ConfigManager.kt

```diff
@@ -0,0 +1,133 @@
+package id.walt.commons.config
+
+import com.sksamuel.hoplite.*
+import io.klogging.java.LoggerFactory
+import java.util.concurrent.ConcurrentLinkedQueue
+import kotlin.reflect.KClass
+import kotlin.reflect.jvm.jvmName
+
+typealias WaltConfig = Any
+
+object ConfigManager {
+
+    val log = LoggerFactory.getLogger(ConfigManager::class.java)!!
+
+    val registeredConfigurations = ConcurrentLinkedQueue<ConfigData>()
+    val loadedConfigurations = HashMap<Pair<String, KClass<out WaltConfig>>, WaltConfig>()
+    val preloadedConfigurations = HashMap<Pair<String, KClass<out WaltConfig>>, WaltConfig>()
```

Comment:
since the e2e tests structure is going to change, is config preloading required?